### ISSUE TITLE
Generator index.ts calls render(App) twice

### ISSUE THREAD

**Issue by LoveSponge** [OfficeDev/generator-office#517]
Issue:

the following code is generated in index.tsx when using this generator:

```js
import "office-ui-fabric-react/dist/css/fabric.min.css";
import App from "./components/App";
import { AppContainer } from "react-hot-loader";
import { initializeIcons } from "office-ui-fabric-react/lib/Icons";
import * as React from "react";
import * as ReactDOM from "react-dom";
/* global AppCpntainer, Component, document, Office, module, require */

initializeIcons();

let isOfficeInitialized = false;

const title = "Contoso Task Pane Add-in";

const render = Component => {
  ReactDOM.render(
    <AppContainer>
      <Component title={title} isOfficeInitialized={isOfficeInitialized} />
    </AppContainer>,
    document.getElementById("container")
  );
};

/* Render application after Office initializes */
Office.initialize = () => {
  isOfficeInitialized = true;
  render(App);
};

/* Initial render showing a progress bar */
render(App);

if ((module as any).hot) {
  (module as any).hot.accept("./components/App", () => {
    const NextApp = require("./components/App").default;
    render(NextApp);
  });
}
```

Where there are two calls to render(App). This causes a race condition, which causes a bug where trying to access `Office.context.mailbox`  returns undefined if office has not been initialized.

I have removed this line from my add-in code and it has fixed the bug, described in more detail, here:
https://github.com/OfficeDev/office-js/issues/980

**TL;DR;**
Remove the synchronous call to render(App) from index.tsx when generating.

```diff
...

/* Render application after Office initializes */
Office.initialize = () => {
  isOfficeInitialized = true;
  render(App);
};

/* Initial render showing a progress bar */
- render(App);

...
```
